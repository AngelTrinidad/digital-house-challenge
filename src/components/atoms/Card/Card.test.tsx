import React from 'react';
import {Text} from 'react-native';

import Card from '.';

import {render} from '~helpers/testing';

describe('<Card />', () => {
  let component: any;

  beforeEach(() => {
    component = render(
      <Card>
        <Text>Hello</Text>
      </Card>,
    );
  });

  it('should render correctly', () => {
    expect(component).toBeDefined();
  });

  it('should has same snapshot', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
