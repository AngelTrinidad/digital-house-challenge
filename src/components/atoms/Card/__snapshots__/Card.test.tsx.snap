// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should has same snapshot 1`] = `
<View
  style={
    [
      {
        "borderRadius": 20,
        "elevation": 11,
        "shadowColor": "black",
        "shadowOffset": {
          "height": 4,
          "width": 0,
        },
        "shadowOpacity": 0.5,
      },
      {
        "backgroundColor": "#334FFA",
      },
      undefined,
    ]
  }
>
  <Text>
    Hello
  </Text>
</View>
`;
